pkg/osquery/table/launcher_identifier.go

```diff
@@ -1,32 +0,0 @@
-package table
```

Comment:
This table has been around for a while, do we want to consider a deprecation cycle?

Reply:
I don't think we need it. This wouldn't remove it from launcher, just from the standalone tables. And I don't think we've official supported that.